modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -67,12 +67,14 @@ func generateWasmCodeHash(code []byte) []byte {
 	return hash[:]
 }
 
-func (k Keeper) storeWasmCode(ctx sdk.Context, code []byte) ([]byte, error) {
+func (k Keeper) storeWasmCode(ctx sdk.Context, code []byte, consumeGas bool) ([]byte, error) {
```

Comment:
it should be possible to do a `ctx = ctx.WithGasMeter(storetypes.NewInfiniteGasMeter())` (or something like that) for genesis

Reply:
it could be possible that baseapp handles this already (?) I can check

Reply:
So that means I can just do the calls to `ConsumeGas` in both cases?

Reply:
Yep! Should be able to remove the bool arg :)

Reply:
Thank you, @damiannolan!